services/web/server/src/simcore_service_webserver/computation_comp_tasks_listening_task.py

```diff
@@ -78,28 +80,53 @@ async def listen(app: web.Application):
             task_output = node_data["outputs"]
             node_id = node_data["node_id"]
             project_id = node_data["project_id"]
+            # FIXME: we do not know who triggered these changes. we assume the user had the rights to do so
+            # therefore we'll use the prj_owner user id. This should be fixed when the new sidecar comes in
+            # and comp_tasks/comp_pipeline get deprecated.
+
             # find the user(s) linked to that project
-            joint_table = user_to_projects.join(projects)
-            query = (
-                select([user_to_projects])
-                .select_from(joint_table)
-                .where(projects.c.uuid == project_id)
+            result = await conn.execute(
+                select([projects]).where(projects.c.uuid == project_id)
             )
-            async for row in conn.execute(query):
-                user_id = row["user_id"]
-                try:
-                    node_data = await projects_api.update_project_node_outputs(
-                        app, user_id, project_id, node_id, data=task_output
-                    )
-                except projects_exceptions.ProjectNotFoundError:
-                    log.exception("Project %s not found", project_id)
-                except projects_exceptions.NodeNotFoundError:
-                    log.exception(
-                        "Node %s ib project %s not found", node_id, project_id
-                    )
-
-                messages = {"nodeUpdated": {"Node": node_id, "Data": node_data}}
-                await post_messages(app, user_id, messages)
+            the_project: RowProxy = result.fetchone()
+            if not the_project:
+                log.warning(
+                    "Project %s was not found and cannot be updated", project_id
+                )
+                continue
+            the_project_owner = the_project["prj_owner"]
+
+            # update the project
+            try:
+                node_data = await projects_api.update_project_node_outputs(
+                    app, the_project_owner, project_id, node_id, data=task_output
+                )
+            except projects_exceptions.ProjectNotFoundError:
+                log.exception(
```

Comment:
Since these are handled, shouldn't we log these as ``debug``? In order to keep the exception info you can always set as
```python
log.debug("Project %s was not found and cannot be updated", project_id, exc_info=True, stack_info=True)
```
This way we don't confuse them with unhandled errors

Reply:
ayayay... I refactor, and now I need to fix the code... I will put a warning, cause this should not happen.